src/features/HelpAndKnowledge/Articles/SixPagerMeeting/index.tsx

```diff
@@ -95,435 +124,568 @@ const SixPagerMeeting = () => {
+              <SimpleList
+                list={conceptPaperSectionSeven}
+                heading={sixPageMeetingT(
+                  'aboutConceptPapers.sectionSeven.heading'
+                )}
+              />
+
+              <SummaryBox className="bg-base-lightest border-0 radius-0 padding-y-2 padding-x-2 margin-y-0">
+                <SummaryBoxHeading headingLevel="h3">
+                  {sixPageMeetingT(
+                    'aboutConceptPapers.additionalResources.heading'
+                  )}
+                </SummaryBoxHeading>
 
-              <div className="margin-bottom-3">
-                <h3 className="margin-top-0 margin-bottom-1">
-                  {sixPageMeetingT('conceptPaper.stepSix.heading')}
-                </h3>
-                <ul className="margin-top-0 margin-bottom-3 padding-left-6">
-                  {stepSix.map(k => (
-                    <li key={k} className="line-height-sans-4">
-                      {k}
-                    </li>
-                  ))}
-                </ul>
-              </div>
+                <SummaryBoxContent className="margin-bottom-1">
+                  <p className="margin-y-1">
+                    <Trans
+                      i18nKey="aboutConceptPapers.additionalResources.description"
+                      t={sixPageMeetingT}
+                      components={{
+                        link1: (
+                          <ExternalLink href="https://share.cms.gov/center/CMMI/SR/ModelDev/Forms/AllItems.aspx">
+                            {' '}
+                          </ExternalLink>
+                        )
+                      }}
+                    />
+                  </p>
+
+                  <div className="margin-y-0 text-bold">
+                    {conceptPaperAdditionalResources.map((section, index) => (
+                      <div key={section.heading}>
+                        {section.heading}
+                        <ul className="margin-top-0 margin-bottom-05 text-normal">
+                          {section.items.map(
+                            (item2: Record<string, string>, index2: number) => (
+                              <li style={{ listStyleType: 'disc' }}>
+                                <Trans
+                                  i18nKey={`aboutConceptPapers.additionalResources.items.${index}.items.${index2}.text`}
+                                  t={sixPageMeetingT}
+                                  components={{
+                                    link1: (
+                                      <ExternalLink
+                                        href={sixPageMeetingT(
+                                          `aboutConceptPapers.additionalResources.items.${index}.items.${index2}.link`
+                                        )}
+                                      >
+                                        {' '}
+                                      </ExternalLink>
+                                    )
```

Comment:
This section has different copy and many missing links, compared to the figma.

### LOCAL: 
![image](https://github.com/user-attachments/assets/e9f23d50-507e-4e35-8cfe-8432bc773e0a)


### FIGMA:
![image](https://github.com/user-attachments/assets/b54e2c93-66c3-4950-81cf-791a385e5cb3)

Reply:
@garyjzhao  Sorry forgot to mention they aren't missing.  Checked with Zoe, she forwards me this updated Figma

Reply:
Looks like she added mroe since she sent me the link